spec/at-rules/nest.md

```diff
@@ -0,0 +1,33 @@
+# `@nest` Rule
```

Comment:
Wasn't this removed?

Reply:
Good catch. Looks like this is an issue in main, so I've put out https://github.com/sass/sass/pull/3913 to fix it.